src/content/docs/journey-app-slow/create-benchmarks.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: 'Create performance benchmarks with service levels'
+metaDescription: 'Learn how to create and define performance benchmarks with service levels.'
+---
+
+import sonicSliUiOverview from 'images/sonic_screenshot-full_sli-ui-overview.png'
+
+## Objectives
+
+By the end of this tutorial, you will:
+
+* Understand service level indicators (SLIs) and service level obectives (SLOs).
+* Have created and defined SLIs and SLOs for your front-end experience.
+* Have alerts set up to warn of any SLI or SLO breaches so you can react before they impact your users.
+
+## Why create performance benchmarks
```

Comment:
IMO given this is a tutorial, it might be better to give a very brief summary <5 sentences about the relevance the task has to their overall goal, but going into detail in a later section (that people can skip if they want). Shuffling content would yield real estate for the beginning of the doc to the actual task, then if they want to know the 'why' they can scroll (or if they don't care, they can just skip)

Reply:
Also it's pretty text heavy too. Another reason to move it imo